test/node/bigint.test.ts

```diff
@@ -263,4 +263,113 @@ describe('BSON BigInt support', function () {
       expect(serializedMap).to.deep.equal(expectedSerialization);
     });
   });
+
+  describe('EJSON.parse()', function () {
+    type ParseOptions = {
+      useBigInt64: boolean | undefined;
+      relaxed: boolean | undefined;
+    };
+    type TestTableEntry = {
+      options: ParseOptions;
+      expectedResult: BSON.Document;
+    };
+
+    const useBigInt64Values = [true, false, undefined];
+    const relaxedValues = [true, false, undefined];
```

Comment:
Can you leave a comment here that we don't need to add "legacy" to the matrix of tests because it does not change the output of a Long. And then to make sure things stay that way two small tests over in long.test.ts that calls fromExtendedJSON with and without legacy set and asserts the result is the same regardless